back/transcendence_django/back_game/game_physics/edges.py

```diff
@@ -0,0 +1,11 @@
+from back_game.game_physics.position import Position
+
+class Edges:
+    def __init__(self, position: Position, width: int, height: int):
+        self.update(position, width, height)
+
+    def update(self, position: Position, width: int, height: int):
+        self.bottom: float = position.y + height / 2
```

Comment:
initialize attributes in the __init__ method and move the types from the update method to the __init__